docs/reference/graph-safe-support.rst

```diff
@@ -0,0 +1,146 @@
+      - `Repository link <https://github.com/ROCm/rocBLAS>`_
+      - ✅
+    * 
+      - hipBLAS
+      - `Repository link <https://github.com/ROCm/hipBLAS>`_
+      - ✅
+    * 
+      - Tensile
+      - `Repository link <https://github.com/ROCm/Tensile>`_
+      - ✅
+    * 
+      - rocFFT
+      - `Repository link <https://github.com/ROCm/rocFFT>`_
+      - ✅
+    * 
+      - hipFFT
+      - `Repository link <https://github.com/ROCm/hipFFT>`_
+      - ✅
+    * 
+      - rocRAND
+      - `Repository link <https://github.com/ROCm/rocRAND>`_
+      - ✅
+    * 
+      - rocPRIM
+      - `Repository link <https://github.com/ROCm/rocPRIM>`_
+      - ✅
+    * 
+      - rocSPARSE
+      - `Repository link <https://github.com/ROCm/rocSPARSE>`_
+      - ⚠️ (experimental)
+    * 
+      - hipSPARSE
+      - `Repository link <https://github.com/ROCm/hipSPARSE>`_
+      - ✅
+    * 
+      - rocHPCG
+      - `Repository link <https://github.com/ROCm/rocHPCG>`_
+      - ❌
+    * 
+      - rocALUTION
+      - `Repository link <https://github.com/ROCm/rocALUTION>`_
+      - ❌
+    * 
+      - MIVisionX
+      - `Repository link <https://github.com/ROCm/MIVisionX>`_
+      - N/A
+    * 
+      - RPP
+      - `Repository link <https://github.com/ROCm/rpp>`_
+      - ⚠️
+    * 
+      - rocAL
+      - `Repository link <https://github.com/ROCm/rocAL>`_
+      - ❌
+    * 
+      - rocDecode
+      - `Repository link <https://github.com/ROCm/rocDecode>`_
+      - ❌
+    * 
+      - rocThrust (:doc:`details <rocthrust:hipgraph-support>`)
+      - `Repository link <https://github.com/ROCm/rocThrust>`_
+      - ❌
+    * 
+      - MIOpen
+      - `Repository link <https://github.com/ROCm/MIOpen>`_
+      - ❌
+    * 
+      - TUNA
+      - `Repository link <https://github.com/ROCm/TUNA>`_
+      - N/A
+    * 
+      - hipCUB
+      - `Repository link <https://github.com/ROCm/hipCUB>`_
+      - ✅
+    * 
+      - rocSOLVER
+      - `Repository link <https://github.com/ROCm/rocSOLVER>`_
+      - ⚠️ (experimental)
+    * 
+      - hipSOLVER
+      - `Repository link <https://github.com/ROCm/hipSOLVER>`_
+      - ⚠️ (experimental)
+    * 
+      - RCCL
+      - `Repository link <https://github.com/ROCm/rccl>`_
+      - ✅
+    * 
+      - rocWMMA
+      - `Repository link <https://github.com/ROCm/rocWMMA>`_
+      - N/A (GPU header library)
+    * 
+      - hipTensor
+      - `Repository link <https://github.com/ROCm/hipTensor>`_
+      - ❌
+    * 
+      - composable kernel
+      - `Repository link <https://github.com/ROCm/composable_kernel>`_
+      - ❌
+    * 
+      - hipBLASLt
+      - `Repository link <https://github.com/ROCm/hipBLASLt>`_
+      - ⚠️
+    * 
+      - hipFORT
+      - `Repository link <https://github.com/ROCm/hipFORT>`_
+      - N/A
+    * 
+      - hipSPARSELt
+      - `Repository link <https://github.com/ROCm/hipSPARSELt>`_
+      - ⚠️ (experimental)
+    * 
+      - rocJPEG
+      - `Repository link <https://github.com/ROCm/rocJPEG>`_
+      - ❌
```

Comment:
For all the components, instead of having a separate column titled "repository" and then having similar entries "repository link", have the repo hyperlink added to the component name in the first column itself. Remove the second column.

Reply:
Removed the second column.